src/modules/trash/trash.usecase.ts

```diff
@@ -20,10 +20,14 @@ export class TrashUseCases {
    * @param trashOwner User whose trash is going to be emptied
    */
   async emptyTrash(trashOwner: User): Promise<void> {
-    const count = await this.fileUseCases.getTrashFilesCount(trashOwner.id);
+    const filesCount = await this.fileUseCases.getTrashFilesCount(
+      trashOwner.id,
+    );
+    const foldersCount = await this.folderUseCases.getTrashFoldersCount(
```

Comment:
It seems like this method does not filter already removed folders. Please do fix it including `{ removed: false }` on the query of folderUseCases.[getTrashFoldersCount](https://github.com/internxt/drive-server-wip/blob/ca4febf2c0978f1617c5c4dc94b0fe050c9a475c/src/modules/folder/folder.usecase.ts#L450)

Reply:
@sg-gs  it was updated in commits after this comment